src/App.js

```diff
@@ -1,16 +1,22 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import AddTodoForm from "./AddTodoForm";
 import TodoList from "./TodoList";
-
+const useSemiPersistentState =()=>{
+  // initialTodoList will get the stored list from localStorage if list dose not exist then it will be initialized with empty array 
+  const initialTodoList=JSON.parse(localStorage.getItem('savedTodoList'))==null?[]:JSON.parse(localStorage.getItem('savedTodoList'));
```

Comment:
Nice job check for null here 💯 
So javascript is a funky language. You can check for truthy value and assign a value to a variable at the same time
For instance, you could write this line of code like this:
```suggestion
  const initialTodoList=JSON.parse(localStorage.getItem('savedTodoList'))==null?[]:JSON.parse(localStorage.getItem('savedTodoList'));
  
  const initialTodoList =  JSON.parse(localStorage.getItem('savedTodoList')) || []
```
Very funky lol So if the localStorage actual has a value `initialTodoList` will equal the saved/persisted array. If it localStorage returns null. Null == false so the `[]` will be assigned to `initialTodoList`

Gotta love JS lol

Reply:
On another note 🎖️ for the `const` :) Nice man!!! Use `const` as much as possible